lib/order.rb

```diff
@@ -0,0 +1,48 @@
+class Order
+
+  def initialize (menu_klass = Menu.new, order_summary_klass = OrderSummary.new, sms_klass = SMS.new)
+    @summary = order_summary_klass
+    @menu = menu_klass
+    @sms = sms_klass
+  end
+
+  def add_item name, quantity = 1
+    raise "Sorry we don't have #{name} on the menu" unless item_on_menu? name
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)